### PR TITLE
Deprecate bidirectional plugin and appendMode

### PR DESCRIPTION
**Pull request by fgalan**
- [x]  deprecate.md
- [x] CNR
- [x] remove bidirectional plugin and appendMode documentation

### CODE REVIEW

**Comment by mapedraza** [comment]
Some notes.

Should this be updated?

https://github.com/telefonicaid/iotagent-node-lib/blob/f71e7385413b7895abb4c0c29ac4d41e75f7c826/doc/northboundinteractions.md?plain=1#L158-L160

https://github.com/telefonicaid/iotagent-node-lib/blob/6535b0935ba8a5307f300447876525272bff6aa0/doc/northboundinteractions.md?plain=1#L603-L604

File: doc/api.md [hunk 1]
```diff
@@ -19,8 +19,7 @@
     -   [Measurement persistence options](#measurement-persistence-options)
         -   [Autoprovision configuration (autoprovision)](#autoprovision-configuration-autoprovision)
         -   [Explicitly defined attributes (explicitAttrs)](#explicitly-defined-attributes-explicitattrs)
-        -   [Configuring operation to persist the data in Context Broker (appendMode)](#configuring-operation-to-persist-the-data-in-context-broker-appendmode)
-        -   [Differences between `autoprovision`, `explicitAttrs` and `appendMode`](#differences-between-autoprovision-explicitattrs-and-appendmode)
+        -   [Differences between `autoprovision`, `explicitAttrs`.](#differences-between-autoprovision-explicitattrs)
```
Comment (fgalan) [inline doc/api.md:22]:
```suggestion
        -   [Differences between `autoprovision`, `explicitAttrs`](#differences-between-autoprovision-explicitattrs)
```

File: doc/api.md [hunk 2]
```diff
@@ -452,29 +446,7 @@ depending on the JEXL expression evaluation:
...
-    "name": "ENTITY_GENERIC_ERROR",
-    "message": "Error accesing entity data for device: deviceType:dev of type: deviceType"
-}
-```
-
-Additionally, the agent creates the device and the corresponding entity in the broker if `autoprovision==true` (default
-behaviour).
-
-### Differences between `autoprovision`, `explicitAttrs` and `appendMode`
+### Differences between `autoprovision`, `explicitAttrs`.
```
Comment (fgalan) [inline doc/api.md:449]:
```suggestion
### Differences between `autoprovision`, `explicitAttrs`
```

**Review by fgalan — COMMENTED** [review]
LGTM

**Pull request merged**